rest_framework_simplejwt/token_blacklist/admin.py

```diff
@@ -67,6 +68,31 @@ class BlacklistedTokenAdmin(admin.ModelAdmin):
         "token__jti",
     )
     ordering = ("token__user",)
+    actions = ["flush_expired_tokens"]
+
+    def flush_expired_tokens(self, request, queryset):
```

Comment:
```suggestion
    @admin.action(permissions=["change"], description = _("Flush expired tokens"))
    def flush_expired_tokens(self, request, queryset):
```
What are your thoughts on using the @admin.action decorator? 
It allows you to set the required permissions for executing an action and provides a description for it as well. Additionally, you can use gettext_lazy to provide localization support. @tpotjj

Reply:
"Can you please execute the 'makemessages' command to generate the 'po' file?"